doc/safety.md

```diff
@@ -0,0 +1,57 @@
+# SECURITY
+
+This document describes the security recommendations and issues that can be helpful for users of Bitcoin Core.
+
+
+Download Bitcoin Core only from below links, verify the integrity of the download before using:
+
+https://bitcoincore.org/en/download/
+
+https://github.com/bitcoin/bitcoin/releases
+
+Downloading Bitcoin Core from untrusted sources can result in using a [malware](https://bitcoin.stackexchange.com/a/107738/) that can steal your bitcoin and compromise your computer.
+
+### P2P
+
+Using non-default ports and non-listening nodes can improve security.
+
+DNS seeds are used for bootstrapping the P2P network. If one of the domains used for default DNS seeds is hacked, this can impact nodes. A node checks 3 seeds at a time, so it would require coordination for a malicious seed to affect the network. You can use own DNS seed for bootstrapping.
+
+Difference in DNS seed and Seed node: https://bitcoin.stackexchange.com/a/17410/
+
+Having multiple connections to the network is fundamental to safety against eclipse attacks. If a user configured the node to only have outbound block-relay-only connections, they wouldn't be participating in addr relay, so wouldn't have a diverse addrman to identify candidates to connect to. It is suggested to have at least 10 outbound connections.
+
+`assumevalid=<hex>` skips validation for scripts. All scripts in transactions included in blocks that are ancestors of the _assumevalid_ block are assumed to be valid. While this does introduce a little bit of trust, people can independently check that block is part of the main chain.
+
+Known incidents when BGP hijacking was used to exploit Bitcoin users: https://www.wired.com/2014/08/isp-bitcoin-theft/
+
+Do not use blocks and chainstate from unknown sources. Attackers could be providing you a false blockchain, i.e. one that is valid but not the blockchain that all other nodes are following.
+
+### RPC
+
+The JSON-RPC interface is intended for local access. By default it will only accept connections from localhost, but it can be configured to be accessed for a wider netmask; e.g. a trusted LAN network. Bitcoin Core does not allow anyone to connect to the rpcport. You will need to explicitly allow an IP address to connect to it by using the `rpcallowip=<ip>` option in the _bitcoin.conf_ file. If you set it to 0.0.0.0, it will be open to all IP addresses, but this is not recommended as it is not secure.
```

Comment:
I don't understand "Bitcoin Core does not allow anyone to connect to the rpcport". Clearly it can be configured to, or RPC would be useless?

Reply:
It can be configured to but does not allow by default. It was copied from this answer: https://bitcoin.stackexchange.com/a/69097/

Reply:
Ok, then say that? The phrase as is is very confusing.